### DISCUSSION TITLE
Next.js app custom Godaddy domain

### DISCUSSION THREAD

**Discussion by denispixi** [vercel/vercel#4471] [Help]
I have a domain in Godaddy and I want it to point to a next.js application deployed in vercel.
When adding the domain, Vercel indicates an error in the configuration of the name servers of my domain and tells me to change the configuration and put the Vercel name servers.
In my domain I have 3 email accounts configured, through the own godaddy.
The question is, do I have to change my name servers ??
Is there no other way to make my domain point to the project without changing that? Something like what is done when you configure a custom domain in Github Pages, adding A and CNAME records.

I hope you can help me, thanks!

**Comment by mcsdevv** [comment]
Hello @denispixi, you can point the domain in one of two ways, by changing nameservers or adding an ANAME record. Changing nameservers is preferable as it gives Vercel more control and allows the platform to potentially deliver increased performance.

ANAME is perfectly fine, however, you can find more information here: https://vercel.com/docs/v2/custom-domains#step-4:-configuring-the-domain

**Reply by denispixi** [reply]
Hi @mcsdevv !! Thanks for the reply 🤟🏽
One more question, my domain ends with the .pe extension and when I try to change the servernames, GoDaddy asks me the server names, but also the IP addresses, and Vercel doesn't give me any IP address for that, just the servernames 
What should I do?.
I think it is strange and related to the extension, because when I try to do the same with another domain ending with .com, no IPs are required, only servernames. What I did was ping the Vercel servernames and use those IPs. Now my domain works fine, but I don't know if what I did (ping and use the obtained IPs) is correct. Do you know what I have to do?

**Comment by j-lee8** [comment]
I added the Vercel nameservers to GoDaddy 16h ago and it's showing a weird message 

`The Sponsored Listings displayed above are served automatically by a third party. Neither Parkingcrew nor the domain owner maintain any relationship with the advertisers.`

Never seen anything like this before. Has it not propagated fully yet?
